src/main/kotlin/com/openlattice/hazelcast/processors/RemoveDataExpirationPolicyProcessor.kt

```diff
@@ -0,0 +1,31 @@
+package com.openlattice.hazelcast.processors
+
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.EntitySet
+import java.util.UUID
+
+class RemoveDataExpirationPolicyProcessor() :
+        AbstractRhizomeEntryProcessor<UUID, EntitySet, EntitySet>() {
+
+    companion object {
+        private const val serialVersionUID = -6602384557982347L //TODO ????
```

Comment:
`TODO` ?

Reply:
i don't remember what that was for... i'll delete it